includes/Core/Authentication/Google_Proxy.php

```diff
@@ -330,6 +347,35 @@ public function get_site_fields() {
 		);
 	}
 
+	/**
+	 * Gets metadata fields.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array Metadata fields array.
+	 */
+	public function get_metadata_fields() {
+		$metadata = array(
+			'supports'         => implode( ',', $this->get_supports() ),
+			'nonce'            => wp_create_nonce( self::ACTION_SETUP ),
```

Comment:
This has to use `self::NONCE_ACTION` for the action per the changes in #4383 and #4376.